lib/ai/tools/deep-research.ts

```
@@ -0,0 +1,381 @@
+    method: 'POST',
+    headers: {
+      Authorization: `Bearer ${process.env.PERPLEXITY_API_KEY}`,
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(payload),
+    signal: abortSignal,
+  });
+
+  if (!response.ok) {
+    const responseBody = await getResponseBody(response);
+    let errorData;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)